doc/lua_api.md

```diff
@@ -5483,6 +5487,9 @@ Utilities
       mod_storage_on_disk = true,
       -- "zstd" method for compress/decompress (5.7.0)
       compress_zstd = true,
+      -- The upper four bits of liquid node param2 values are preserved
+      -- during liquid flow (5.10.0)
+      preserve_liquid_param2 = true,
```

Comment:
It should be on the End, not between 5.7.0/5.8.0 features.
Also in `features.lua`.